src/network/networkd-routing-policy-rule.h

```diff
@@ -34,6 +34,7 @@ typedef struct RoutingPolicyRule {
         int family; /* Automatically determined by From= or To= */
         unsigned char to_prefixlen;
         unsigned char from_prefixlen;
+        unsigned char type;
```

Comment:
Please also set to RTN_UNICAST in `routing_policy_rule_new()`.

Reply:
kernel default ? Where it's set ?

Reply:
`sd_rtnl_message_new_routing_policy_rule()` now sets `FR_ACT_TO_TBL` by default. So, please also set the same value in `routing_policy_rule_new()`. Otherwise, all created rules are handled as 'foreign'.

Reply:
Yes the whole point if migrate it to do this.